src/main/scala/org/broadinstitute/dsde/agora/server/webservice/ApiServiceActor.scala

```diff
@@ -55,11 +57,32 @@ class ApiServiceActor extends HttpServiceActor with LazyLogging {
   val methodsService = new MethodsService() with ActorRefFactoryContext
   val configurationsService = new ConfigurationsService() with ActorRefFactoryContext
 
+  def withResourceFileContents(path: String)(innerRoute: String => Route): Route =
+    innerRoute( FileUtils.readAllTextFromResource(path) )
 
   def possibleRoutes =  options{ complete(OK) } ~ methodsService.routes ~ configurationsService.routes ~
     get {
       pathSingleSlash {
-        getFromResource("swagger/index.html")
+        withResourceFileContents("swagger/index.html") { indexHtml =>
+          complete {
+            val swaggerOptions =
+              """
+                |        validatorUrl: null,
+                |        apisSorter: "alpha",
+                |        operationsSorter: "alpha",
+              """.stripMargin
+            HttpEntity(ContentType(MediaTypes.`text/html`),
+              indexHtml
+                .replace("your-client-id", AgoraConfig.SwaggerConfig.clientId)
+                .replace("your-realms", AgoraConfig.SwaggerConfig.realm)
+                .replace("your-app-name", AgoraConfig.SwaggerConfig.appName)
+                .replace("scopeSeparator: \",\"", "scopeSeparator: \" \"")
+                .replace("jsonEditor: false,", "jsonEditor: false," + swaggerOptions)
```

Comment:
also, this line isn't going to work - `jsonEditor` was added in a later version of swagger-ui. You'll have to either upgrade swagger-ui, or find something else to hang the replacement off of.